style.css

```diff
@@ -60,6 +60,7 @@ nav > a {
 }
 
 .profile > h1 {
+  margin-top: 128px;
   line-height: 52px;
   color: #172b4d;
 }
```

Comment:
- [ ]  Hey, although this project has been approved, according to the reviewer's comment, it's because it was your last review, so he marked it as optional. Just so you know, the changes below are now required and should be implemented as much as possible to improve your design.

- Let's improve the space on the background image as yours is currently not consistent when compared to the Figma design. You added the background image to the body and now it will be tricky for you to move it down below the header. Please, kindly remove the background from the body and add it to the section `profile` so you can adjust the padding, margin-top to control the spacing as seen below:

*Figma design*
![image](https://github.com/santitoot/1-project-mobil-final-/assets/59397159/937c8490-4ada-4813-ad41-26d0e826bd2b)


*Your design*
![image](https://github.com/santitoot/1-project-mobil-final-/assets/59397159/d6334448-293a-47b0-b7b9-6dd11bd3f270)

Reply:
I made the required change